eo-maven-plugin/src/main/java/org/eolang/maven/BinarizeParseMojo.java

```diff
@@ -204,29 +139,26 @@ private XML addRust(
     }
 
     /**
-     * Makes a text from Hexed text.
-     * @param txt Hexed chars separated by backspace.
-     * @return Normal text.
+     * Add ffi node via xsl transformation and return list of them.
+     * @param input Input xmir.
+     * @return FFI nodes.
+     * @checkstyle AbbreviationAsWordInNameCheck (8 lines)
      */
-    private static String unhex(final String txt) {
-        final StringBuilder hex = new StringBuilder(txt.length());
-        for (final char chr : txt.toCharArray()) {
-            if (chr == ' ') {
-                continue;
-            }
-            hex.append(chr);
-        }
-        final String result;
-        try {
-            final byte[] bytes = Hex.decodeHex(String.valueOf(hex).toCharArray());
-            result = new String(bytes, StandardCharsets.UTF_8);
-        } catch (final DecoderException exception) {
-            throw new IllegalArgumentException(
-                String.format("Invalid String %s, cannot unhex", txt),
-                exception
+    private Collection<FFINode> getFFIs(final XML input) {
```

Comment:
@levBagryansky let's make this class more generic and unified in order to be able to add more FFIs (with C language for example). It means that `addRust` method should not be here because it's concrete and if we would want to add FFI with C language we would need to add `addC` method. And the same story for all other potential FFIs. Or you can just add a puzzle.

Reply:
@maxonfjvipon Maybe leave a puzzle here?

Reply:
@levBagryansky sure